apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx

```diff
@@ -0,0 +1,26 @@
+import Stand from "./Stand";
+import stand1 from "./stand-1.png";
+import stand2 from "./stand-2.png";
+
+const MentorVolunteer = () => {
+	return (
+		<div className="grid max-w-lg lg:max-w-none lg:grid-cols-[repeat(2,450px)] lg:items-stretch justify-center m-auto gap-y-8 gap-x-24 mt-24">
+			<Stand
+				imageSrc={stand1}
```

Comment:
Issue: On mobile views, the stands aren't centered. We should also add some additional margin to the sides.

<img width="161" alt="image" src="https://github.com/HackAtUCI/irvinehacks-site-2024/assets/28026520/5d6804bf-8e47-4731-a534-6be2987c63ad">

Reply:
The off centered-ness is mostly baked into the asset itself. I've added an uneven padding on the stand section to counteract this and add spacing on the mobile view.